### ISSUE TITLE
Adding SQL Resources

### ISSUE THREAD

**Issue by eriktrautman** [TheOdinProject/curriculum#1636]
TLDR: New resource available.  We should add Khan Academy's new course as well.

****
Hey,

First of thank you for this simply amazing collection of resources and structuring them in the way that you did! However I was not very satisfied with the SQL side of things.

The DB1 and DB5 courses picked up pace way to quickly without enough explanation and I found the SQL Zoo to not be very helpful at all.

I suggest that you change the primary SQL resource to http://sqlbolt.com it's a very good interactive tutorial very similar to the RegEx ones.

Let me know what you think!

Kind Regards,

Alexander

**Comment by nhquiroz** [comment]
This quick overview about SQL from CS50 can be useful too: https://www.youtube.com/watch?v=G58ujNjWEJY

**Comment by lpatmo** [comment]
I found the Stanford course excellent and much more advanced than Khan Academy's, which was a great introduction, but didn't touch on more complicated queries: 

https://lagunita.stanford.edu/courses/DB/SQL/SelfPaced/courseware/ch-sql/seq-exercise-sql_movie_query_core/

**Comment by KevinMulhern** [comment]
We made the khan academy course the main resource for the database course a while back. This should have been closed 😄
